web/src/components/SharingPopup.tsx

```diff
@@ -192,6 +196,11 @@ const SharingPopup = ({ shareUrl, title, flow, portalNeeded }: SharingPopupProps
   const encodedShareUrl = encodeURIComponent(shareUrl)
   const shareMessage = t('layout:shareMessage', { message: encodedTitle })
 
+  const { viewportSmall } = useWindowDimensions()
+  const theme = useTheme()
+  const tooltipDirectionMobile: PlacesType = theme.contentDirection === 'ltr' ? 'right' : 'left'
+  const tooltipDirection: PlacesType = viewportSmall ? tooltipDirectionMobile : 'top'
```

Comment:
:upside_down_face: we could still make use of a custom tooltip component to avoid duplicating this and other things:
```tsx
import React, { ReactElement } from 'react'
import { Tooltip as ReactTooltip, PlacesType } from 'react-tooltip'
import { useTheme } from 'styled-components'

import useWindowDimensions from '../../hooks/useWindowDimensions'

type TooltipProps = {
  children: ReactElement
  text: string
  place?: PlacesType
  id: string
}

const Tooltip = ({ children, text, id, place }: TooltipProps): ReactElement => {
  const { viewportSmall } = useWindowDimensions()
  const theme = useTheme()
  const tooltipDirectionMobile: PlacesType = theme.contentDirection === 'ltr' ? 'right' : 'left'
  const tooltipDirection: PlacesType = viewportSmall ? tooltipDirectionMobile : 'top'

  return (
    <>
      <div id={id}>{children}</div>
      <ReactTooltip anchorSelect={`#${id}`} place={place ?? tooltipDirection}>
        {text}
      </ReactTooltip>
    </>
  )
}

export default Tooltip
```
I think this would simplify the usage and prevent mistakes by misspelling the ids.
Completely optional tough.